cmd/siac/export.go

```diff
@@ -30,13 +30,12 @@ var (
 // renterexportcontracttxnscmd is the handler for the command `siac renter export contract-txns`.
 // Exports the current contract set to JSON.
 func renterexportcontracttxnscmd(destination string) {
-	cs, err := httpClient.RenterContractsGet()
+	cs, err := httpClient.RenterContractsGet(false)
```

Comment:
since it's not obvious at all what `true` means in the context of this function, convention is to use a variable:

```
getAllContracts := true
rc, err := httpCliend.RenterContractsGet(getAllContracts)
```